Makefile.pre.in

```diff
@@ -2759,6 +2762,14 @@ patchcheck: all
 check-limited-abi: all
 	$(RUNSHARED) ./$(BUILDPYTHON) $(srcdir)/Tools/build/stable_abi.py --all $(srcdir)/Misc/stable_abi.toml
 
+.PHONY: bundle-ensurepip-dists
+bundle-ensurepip-dists: $(BUILDPYTHON) platform
```

Comment:
ISTM this target name should explicitly call out that it is (probably) downloading something from the internet, something more along the lines of `download-ensurepip-blobs`.

Reply:
Fair point. I like this suggestion over https://github.com/python/cpython/pull/12791#discussion_r1306868995.

Reply:
I've applied this change, so I'm marking the thread as resolved.